<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddressBookFrame</class>
 <widget class="QFrame" name="AddressBookFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="m_addressBookView">
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="headerDefaultSectionSize">
      <number>150</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="m_addAddressButton">
       <property name="text">
        <string>New contact</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/add_recepient</normaloff>:/icons/add_recepient</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_editAddressButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Edit contact</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/edit</normaloff>:/icons/edit</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_deleteAddressButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Delete contact</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_copyAddressButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Copy address</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/copy_white</normaloff>
         <selectedoff>:/icons/copy_white</selectedoff>:/icons/copy_white</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="m_copyPaymentIdButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Copy Payment ID</string>
       </property>
       <property name="icon">
        <iconset resource="../../resources.qrc">
         <normaloff>:/icons/copy_white</normaloff>
         <selectedoff>:/icons/copy_white</selectedoff>:/icons/copy_white</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>m_addAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>addClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>53</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_deleteAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>deleteClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>148</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>copyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>556</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_editAddressButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>editClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_copyPaymentIdButton</sender>
   <signal>clicked()</signal>
   <receiver>AddressBookFrame</receiver>
   <slot>copyPaymentIdClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>436</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addClicked()</slot>
  <slot>deleteClicked()</slot>
  <slot>copyClicked()</slot>
 </slots>
</ui>
